pkg/hub/csr/controller.go

```diff
@@ -163,16 +165,3 @@ func isSpokeClusterClientCertRenewal(csr *certificatesv1beta1.CertificateSigning
 
 	return csr.Spec.Username == x509cr.Subject.CommonName
 }
-
-// Check whether a CSR is in terminal state
-func isCSRInTerminalState(status *certificatesv1beta1.CertificateSigningRequestStatus) bool {
```

Comment:
nit, my personal preference is to expose from point of use and then directly use them in tests.  helper package grow weird deps which spider out